packages/fms-base/conf/Dockerfile.j2

```diff
@@ -0,0 +1,37 @@
+{% if arch %}
+FROM base-builder:{{ codename }}-{{ arch }}-latest
+{% else %}
+FROM base-builder:{{ codename }}-latest
+{% endif %}
+LABEL maintainer="{{ vendor}} <{{ vendor_email }}>"
+
+RUN apt-get -qq update && apt-get -qq -y install perl postgresql
+
+WORKDIR /home/builder
+USER builder
+
+RUN git clone --recursive https://github.com/mysociety/fixmystreet.git
+WORKDIR /home/builder/fixmystreet
+RUN git checkout {{ commit }}
+RUN echo "SHORT_COMMIT=$(git rev-parse --short HEAD)" >> /home/builder/.env
+RUN echo "TIMESTAMP=$(date +%Y%m%d%H%M)" >> /home/builder/.env
+
+USER root
+RUN cp conf/packages.docker /opt/
+RUN xargs -a "/opt/packages.docker" apt-get -qq -y install >/dev/null
+
+USER builder
+RUN ./script/bootstrap
+RUN echo "SHORT_COMMIT=$(git rev-parse --short HEAD)" >> /home/builder/.env
+RUN fpm -s dir -t deb \
+      --vendor "mySociety" \
+      --maintainer "mySociety <sysadmin@mysociety.org>" \
+      --version "{{ version }}+git{{ timestamp }}.${SHORT_COMMIT}" \
```

Comment:
`<version>+git<timestamp>.<commit>` is the format for the Debian package version. The thinking here is that using timestamps allows us to publish packages for new commits for a version, rather than only publishing on a new FMS version release.

Reply:
Just pondering this...

Currently `version` is set manually; I think this should always be the most recent tag, so perhaps we're better deriving this based on what we check out with `git describe --tags`? [Debian package version numbers also need to start with a digit](https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-version), so we'd have to strip the leading `v`.

So then we could just specify `commit`, which actually could be anything that we can check out (so a branch, tag, commit, etc). Is there a better term to use or is this OK?

I think a timestamp of YYYYMMDD is probably fine, given we will include a commit hash as well.

We'd end up with something like `6.0+git20250319.d3c4fb4d`.

Reply:
Probably unlikely, but not impossible. Can retain HHMM, see also discussion on version/iteration.